### DISCUSSION TITLE
No idea what I'm doing wrong, but can not install

### DISCUSSION THREAD

**Discussion by rzaprianov** [ghostery/user-agent-desktop#586] [Q&A]
Hello,
I made everything until the last step described in FAQ for install under Linux. I have Ubuntu 20.04 LTS 64bit. But last point with the script I make any mistake (according to me) https://i.imgur.com/j28fZxr.png

Here it is:
https://i.imgur.com/Z2J6vMB.png

And nothing happen after that - the browser does not appear anywhere.

**Comment by philipp-classen** [comment]
Hi, we are working on improving the documentation. In the meantime, can you please try the following:

```
$ cat > ~/bin/ghostery <<END
#!/bin/bash

exec "\$HOME/Ghostery/Ghostery” -P ghostery-build "\$@"
END
```

Note that the last line with END was missing in the docs. After running it, you should have the file, and it should look like that:

```
$ cat ~/bin/ghostery 
#!/bin/bash

exec "$HOME/Ghostery/Ghostery” -P ghostery-build "$@"
```

Now, you can try to start Ghostery Dawn:
```
$ ~/bin/ghostery
```

Hope that helps.

**Comment by rzaprianov** [comment]
Hi there! Thanks for your answer!

It works now
https://i.imgur.com/Qkas5C5.jpg


One more question - in the other browsers I used I have a script on a 
web page based on jquery.isotope.js that arrange icons with links but 
here it doesn't works - missing functionality or function I have to turn 
on?


Regards,


Rumen



На 10.01.22 г. в 12:57, Philipp Claßen написа:

**Comment by philipp-classen** [comment]
Great to hear that you were able to install. Regarding your follow-up question, I'm not sure if I fully understood it. Do you mean, Ghostery does not show up when search for applications to start?

If it is that, then it could be because the current installation does not setup a `.desktop` file. What we should do is to put a file in `/usr/share/applications/` or `~/.local/share/applications`. For instance, Firefox does that:

```
$ find /usr/share/applications/ | grep firefox
/usr/share/applications/firefox.desktop
```

Could it be related to what you described?

**Comment by rzaprianov** [comment]
No, look at the attached file. It works in Firefox, Chrome, Chromium and 
Yandex but not in Ghostery here is the address - as I have a lot of 
pages I use, I made a script that I can call from any browser anywhere I 
go .... https://ucc-bg.com/1st_page/ and it sort (filter) the pages by 
topics. If it is a matter of function that I should turn on I will 
turnit on. If not I will use some other way to sort the pages :)


1 файл е свързан с този мейл:
vokoscreenNG-2022-01-10_16-10-18.mkv 
<https://zapryanov.faith/cloud/index.php/s/f3f9wEDrRdfxKX2/download>(7,4 
MB)*cloud 
<https://addons.thunderbird.net/de/thunderbird/addon/filelink-nextcloud-owncloud/>https://zapryanov.faith/cloud/index.php/s/f3f9wEDrRdfxKX2/download 
<https://zapryanov.faith/cloud/index.php/s/f3f9wEDrRdfxKX2/download>
Mozilla Thunderbird <https://www.thunderbird.net> прави лесно 
споделянето на големи файлове чрез мейл.


На 10.01.22 г. в 16:00, Philipp Claßen написа:

**Comment by philipp-classen** [comment]
Ah, OK. Unfortunately, I cannot help with that. Still, I hope that you can find a way that works for you.
